config/locales/js-en.yml

```diff
@@ -1455,4 +1455,4 @@ en:
         network_on: "Network is back. We are trying."
         timeout: >
           OpenProject could not provide you access to the project folder within the expected period of time. Please, try once again.<br><br>
-          If that problem persists please contact your OpenProject administrator to check the <a href="https://www.openproject.org/docs/system-admin-guide/files/external-file-storages/file-storage-troubleshooting/">health status of the file storage setup</a>.
+          If that problem persists please contact your OpenProject administrator to check the <a href="https://www.openproject.org/docs/system-admin-guide/files/external-file-storages/health-status/">health status of the file storage setup</a>.
```

Comment:
I can fix this in other languages as well, but I am not sure if this affects crowdin process. What would be preferred to do?

Reply:
This way, the localisation does not have to be updated, if the link is changing

Reply:
Nah, I think this is not as difficult. This is a stimulus controller, hence you can pass values in. Like it is done with `projectStorageOpenUrlValue` and `redirectUrlValue`.

And those values are built in rails context.

Reply:
Fair enough. Lemme try, then :sweat_smile: